config/de/mkdocs.yml

```diff
@@ -389,7 +389,6 @@ nav:
   - Auswertungen:
       - auswertungen/index.md
       - Report-Manager: auswertungen/report-manager.md
-      - Variable Reports: auswertungen/variable-reports.md
```

Comment:
If you remove or move a page please create a redirect entry around line 64
'old path': 'new path'